ppr-ui/src/components/mhrTransfers/SupportingDocuments.vue

```diff
@@ -48,22 +48,21 @@ import { ApiTransferTypes, SupportingDocumentsOptions } from '@/enums/transferTy
 import { MhrRegistrationHomeOwnerIF } from '@/interfaces' // eslint-disable-line no-unused-vars
 import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
 import { useActions } from 'vuex-composition-helpers'
-import { DeathCertificate } from '.'
 import { transferSupportingDocuments } from '@/resources/'
 
 export default defineComponent({
   name: 'SupportingDocument',
   props: {
     deletedOwner: {
       type: Object as () => MhrRegistrationHomeOwnerIF,
-      default: null
+      required: true
     },
     // Used to disable Death Cert when group has only one owner
     isSecondOptionDisabled: {
       type: Boolean
     }
   },
-  components: { DeathCertificate },
+  components: { },
```

Comment:
Do we need an empty components here?

Reply:
We definitely do not.
I'm guilty of leaving these in, in a few spots. I thought these component options make adding/editing a component more intuitive, like an empty `emits: []` as another example.

I'm 100% ok with removing though too, whatever you prefer!